readme.md

```diff
@@ -125,7 +122,7 @@ Launch the local HTTPS site on `https://localhost:3000` by simply typing the fol
 $ npm start
 ```
 
-> **Note**: Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run `npm start`, accept the prompt to install the certificate that the Yeoman generator provides. For more information about enabling HTTPS for your add-in, see [Installing the self-signed certificate](src/docs/ssl.md).
+> **Note**: Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run `npm start`, accept the prompt to install the certificate that the Yeoman generator provides.  For more information, see [Installing the self-signed certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md).

```

Comment:
no longer reference local src/docs file here

Reply:
You need to delete the sentence "For more information, see...  ssl.md)"